docs/install/setup-aws.rst

```diff
@@ -0,0 +1,79 @@
+.. _aws-install:
+
+========================================================
+Setting Up Flocker-Ready Nodes Using Amazon Web Services
+========================================================
+
+.. note:: If you are not familiar with EC2 you may want to `read more about the terminology and concepts <https://fedoraproject.org/wiki/User:Gholms/EC2_Primer>`_ used in this document.
+          You can also refer to `the full documentation for interacting with EC2 from Amazon Web Services <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html>`_.
+
+
+.. The AMI links were created using the ami_links tool in ClusterHQ's internal-tools repository.
+
+#. Choose a nearby region and use the link to it below to access the EC2 Launch Wizard.
+   These launch instances using CentOS 7 AMIs (in particular "CentOS 7 x86_64 (2014_09_29) EBS HVM") but it is possible to use any operating system supported by Flocker with AWS.
```

Comment:
:question: "it is possible to use any operating system supported by Flocker with AWS" - maybe link to this list